content/pt/case-studies/chinaunicom/index.html

```diff
@@ -0,0 +1,96 @@
+<br><br>
+    <h2>Solução</h2>
+    Por causa de seu crescimento rápido e comunidade madura de código aberto, o Kubernetes foi uma escolha natural para a China Unicom. A plataforma de nuvem habilitada para Kubernetes da empresa agora hospeda 50 microsserviços e todo o novo desenvolvimento daqui para frente. "O Kubernetes melhorou nossa experiência usando a infraestrutura de nuvem", diz Zhang. "Atualmente, não há tecnologia alternativa que possa substituí-lo." A China Unicom também usa o <a href="https://istio.io/">Istio</a> para sua estrutura de microsserviço, <a href="https://www.envoyproxy.io/">Envoy</a>, <a href="https://coredns.io/">CoreDNS</a> e <a href="https://www.fluentd.org/">Fluentd</a>.
+<br><br>
+<h2>Impacto</h2>
+    Na China Unicom, o Kubernetes melhorou a eficiência operacional e de desenvolvimento. A utilização de recursos aumentou de 20 a 50%, diminuindo os custos de infraestrutura de TI, e o tempo de implantação passou de algumas horas para 5 a 10 minutos. "Isso se deve principalmente à autocorreção e à escalabilidade, para que possamos aumentar nossa eficiência em operação e manutenção", diz Zhang. "Por exemplo, atualmente temos apenas cinco pessoas mantendo nossos múltiplos sistemas. Nós nunca poderíamos imaginar que podemos alcançar essa escalabilidade em tão pouco tempo".
+
+</div>
+
+</div>
+</section>
+<div class="banner2">
+  <div class="banner2text">
+    "O Kubernetes melhorou nossa experiência usando a infraestrutura em nuvem. Atualmente, não há tecnologia alternativa que possa substituí-lo."
+<br style="height:25px"><span style="font-size:14px;letter-spacing:2px;text-transform:uppercase;margin-top:5% !important;"><br>- Chengyu Zhang, Líder do Grupo de Pesquisa e Desenvolvimento em Tecnologia de Plataforma, China Unicom</span>
+  </div>
+</div>
+<section class="section2">
+<div class="fullcol">
+  <h2>Com mais de 300 milhões de usuários, a China Unicom é uma das três principais operadoras de telecomunicações do país. </h2>
+
+  Nos bastidores, a empresa administra vários datacenters com milhares de servidores em cada um, usando contêineres Docker e infra-estrutura VMWare e OpenStack desde 2016. Infelizmente, "a taxa de utilização de recursos foi relativamente baixa", diz Chengyu Zhang, líder de pesquisa e desenvolvimento de plataformas tecnológicas. "e não tínhamos uma plataforma de nuvem para acomodar nossas centenas de aplicativos". <br> <br>
```

Comment:
You can remove the comma here as well ->

`usando contêineres Docker e infra-estrutura VMWare e OpenStack desde 2016.`

Reply:
Felipe, Remove or added? Got confused...

Reply:
I'm, sorry, add the comma :)